.gitignore

```diff
@@ -15,3 +15,4 @@
 /log/*.log
 /tmp
 .env
+logfile
```

Comment:
I think this may have been the result of a command I ran. I'm going to remove from the .gitignore for now.